ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -84,7 +81,7 @@ function createMap(insertionElement) {
       animation: false,
       fullscreenButton: false,
       timeline: false,
-      geocoder: new gazetteer(),
+      geocoder: false,
```

Comment:
Why is this false and not null/undefined, or this line even removed?

Reply:
This disables the Cesium geocoder. We don't need it anymore since we manually mount our own.